app/views/components/promotion.html

```diff
@@ -30,7 +30,11 @@
           ></i>
         </button>
       </div>
-      <div class="col-md-5 col-md-pull-3 stacked-spacing-col-md">
+
```

Comment:
This is a huge improvement! The first row of inputs almost lines up with the chartfield inputs, but not quite. I'm probably OCD, but it bothers me and might bother users too. Could you try to get them to line up?